### ISSUE TITLE
Some small issues

### ISSUE THREAD

**Issue by laanwj** [henck/korenvliet#1]
Fun to play an IF game situated in the Netherlands :)
Reporting some small issues I encountered while playing.

Bugs and typos:

- Plateau: shows no exits
- Abandoned barn:
   - "plateau" is not recognized even though mentioned in description. This one tends to be quite common,  that objects are mentioned in descriptions but not recognized, but only mentioning it once.
   - "abanboned" typo in description of cobwebs
- "In 498 moves, you have scored 8 of a possible 7 points."

A bit confusing:

In the beginning I had no idea how to use the boat. I tried
to put it in the lake:
```
>put boat in lake
You can't put anything in the lake.
```
And in the brook, with the same outcome.

**Comment by henck** [comment]
Many thanks! I have implemented fixes:

* Plateau now shows exits.
* In abandoned barn, fixed typo and plateau is now a recognized exit. Added some more decorations here and there.
* Fixed total score, which is indeed 8 points.
* Added better messages for putting the boat in the lake / brook.

The original Korenvliet was a very bare-bones adventure, with no descriptions for anything. I've added many descriptions, but I agree that there are still many objects mentioned in the room descriptions that cannot be examined. It would be better to add many more scenery descriptions.
